libcxx/test/benchmarks/containers/associative/associative_container_benchmarks.h

```diff
@@ -172,6 +172,21 @@ void associative_container_benchmarks(std::string container) {
     }
   });
 
+  bench("operator=(const&) (into populated Container)", [=](auto& st) {
```

Comment:
I would suggest adding coverage for assigning into a map that has something like half the nodes of the incoming map. That would help cover the case of assigning between trees that have a different structure.